### ISSUE TITLE
Add opus testcase

### ISSUE THREAD

**Issue by prudhomm** [feelpp/toolbox#10]
The opus testcase needs to be added to the toolbox benchmarks, we have some results to compare with in some cases (with Comsol)

**Comment by prudhomm** [comment]
@GQMP  I would like you to be on the forefront of these developments.

**Comment by GQMP** [comment]
Where can I download the reference results ?

**Comment by prudhomm** [comment]
- [OpusBenchmarkValidation.pdf](https://github.com/feelpp/toolbox/files/1974281/OpusBenchmarkValidation.pdf)
I have also the tex files and everything I think

**Comment by prudhomm** [comment]
I just shared with you the opus benchmark via dropbox.
you can use pandoc to transform tex files into asciidoc files

**Comment by prudhomm** [comment]
@gqmp any news on your front?

**Comment by jbwahl** [comment]
@prudhomm j'avais un pdf plus complet sur Opus que tu m'avais fourni. Est-ce volontaire de ne pas l'avoir transmis ?  Est-ce que je peux le partager ici ?

**Comment by jbwahl** [comment]
@prudhomm can you share the dropbox folder with me ?

**Comment by prudhomm** [comment]
I provided the full opus access via dropbox

**Comment by prudhomm** [comment]
there is also a dataset to compare the results with comsol

**Comment by jbwahl** [comment]
thx

**Comment by GQMP** [comment]
@prudhomm I did the tests, and I try to extract the results with Paraview.
For the stationary case, I already have the results : mean of 338.38 °K on IC2 and mean of 312.86 °K  on the output, but the reference results are lower (respectively 322.75 °K and 310.42 °K). It might be the difference of the boundary conditions on sides (adiabatic vs periodic), but then I don't know how to compare correctly the data.

**Comment by prudhomm** [comment]
Yes that's it. No problem with that. We need to support two things that were supported in the past :
* periodic boundary conditions
* locally discontinuous functions

**Comment by prudhomm** [comment]
The overhaul profile of the outputs should be similar though, no?

**Comment by prudhomm** [comment]
@GQMP where do we stand on this project?

**Comment by prudhomm** [comment]
@GQMP where do we stand on this ? @romainhild  @jbwahl  ?

**Comment by prudhomm** [comment]
where are the plot.ly graph ?
http://docs.feelpp.org/examples/wip-example-3-opuseads-heat/heat/opus/README/

**Comment by prudhomm** [comment]
@GQMP I just accepted the PR, could you update the testcase ?

**Comment by prudhomm** [comment]
This was not finished it seems see #119 
@GQMP could you do it ? if you have any questions please ask
 - add images
 - add video with NS flow + heat transfer in time.

**Comment by GQMP** [comment]
@prudhomm I modified the file toolbox/supplemental-ui/partials/head-meta.hbs 2 times (in the past), and it seems that the web page use the first version and not the second. As the second version is on all the branches of feelpp/toolbox, I suppose the site is not generated from this repository.

**Comment by GQMP** [comment]
It seems that I forgot to make a pull request on book.feelpp.org...

**Comment by prudhomm** [comment]
Ok and how about the testcase? could you finish it?

**Comment by GQMP** [comment]
I don't know why but the page http://docs.feelpp.org/examples/wip-example-3-opuseads-heat/heat/opus/README/ shouldnt exists anymore, with the right antora module the address is now http://docs.feelpp.org/cases/0.106/heat/opus/README/.

**Comment by GQMP** [comment]
@prudhomm I have the heat images of all the tests, and the video for the case test1.
